src/compiler/output-targets/dist-custom-elements/index.ts

```diff
@@ -104,7 +104,7 @@ export const getBundleOptions = (
  */
 
 export const bundleCustomElements = async (
-  config: d.Config,
+  config: d.ValidatedConfig,
```

Comment:
Similar to my comment above, can we update the JSDoc `@param` description for `config` here as well?
```diff
- * @param config user-supplied Stencil configuration
+ * @param config the validated compiler configuration we're using
```

It also looks like this JSDoc is 'floating' with a newline between it and the function. While you're in there, could you remove that pesky extra `\n` too please?